### ISSUE TITLE
How to convert a simple handler to JettyXML

### ISSUE THREAD

**Issue by a1730** [eclipse/jetty.project#10917]
**Jetty Version**
 jetty-12.0.3; built: 2023-10-27T03:25:22.284Z;

**Jetty Environment**
Core

**Java Version**
openjdk version "21" 2023-09-19 LTS jvm 21+35-LTS

**Question**
I have a simple jetty handler that works very well in embedded mode and decided to use the opportunity to learn JettyXML IOC.
Things work well for line-by-line conversion of the handler but for these lines
```java
    // Create and configure a ResourceHandler.
    final ResourceHandler assets = new ResourceHandler();
    // Configure the directory where static resources are located.
    assets.setBaseResource(ResourceFactory.of(assets).newClassLoaderResource("assets"));
```
Jetty XML throws an exception about newClassLoaderResource.
The following is one of my many, many attempts
```xml
  <New id="resourceFactory" class="org.eclipse.jetty.util.resource.PathResourceFactory">
    <Call name="newClassLoaderResource" arg="assets"/>
  </New>
```
What am I doing wrong?
Thank you.

**Comment by sbordet** [comment]
Stack trace of the exception?

**Comment by a1730** [comment]
I figured it out:)
The insight came from an unusual place [sbordet:jetty-11-http-1-1-to-http-2-upgrade-how-to-write-the-handler](https://stackoverflow.com/a/68405248). There are multiple layers of abstraction provided by Jetty (Thank you!), so understanding where I was in the hierarchy allowed me to pass my custom handler to the context in JettyXML, and boom, it works. i.e. I created a very simple handler in java, and passed it to JettyXML => Life is good.

Thank you very much for the quick response.
